src/sync/discovery.ts

```diff
@@ -56,4 +89,4 @@ export const discovery = (app: Application) => async (job, logger: LoggerWithTar
 	} catch (err) {
 		logger.error({ job, err }, "Discovery error");
```

Comment:
🤔 I've just realised if any error happens here the job won't be retried. Not sure why it was there, but I think we need to remvoe this try/catch block, so the retry will be properly handled by sqs listener